client/post-editor/editor-ground-control/index.jsx

```diff
@@ -71,57 +72,6 @@ export class EditorGroundControl extends PureComponent {
 		setPostDate: noop,
 	};
 
-	state = {
-		needsVerification:
-			this.props.userUtils && this.props.userUtils.needsVerificationForSite( this.props.site ),
-	};
-
-	componentDidMount() {
-		if ( ! this.props.user ) {
-			return;
-		}
-
-		this.props.user
-			.on( 'change', this.updateNeedsVerification )
-			.on( 'verify', this.updateNeedsVerification );
-	}
-
-	componentWillUnmount() {
-		if ( ! this.props.user ) {
-			return;
-		}
-
-		this.props.user
-			.off( 'change', this.updateNeedsVerification )
-			.off( 'verify', this.updateNeedsVerification );
-	}
-
-	updateNeedsVerification = () => {
-		this.setState( {
-			needsVerification:
-				this.props.userUtils && this.props.userUtils.needsVerificationForSite( this.props.site ),
-		} );
-	};
-
-	componentWillReceiveProps( nextProps ) {
-		this.setState( {
-			needsVerification:
-				nextProps.userUtils && nextProps.userUtils.needsVerificationForSite( nextProps.site ),
-		} );
-
-		if ( this.props.user ) {
-			this.props.user
-				.off( 'change', this.updateNeedsVerification )
-				.off( 'verify', this.updateNeedsVerification );
-		}
-
-		if ( nextProps.user ) {
```

Comment:
this `componentWillReceiveProps` was causing a lot of thrashing with subscribing/unsubscribing to `user` on every render!